spec/create_spec.rb

```diff
@@ -124,4 +124,39 @@
 
   end
 
+  context 'validation for AWS credentials' do
+
+    context 'IAM profile is not used' do
+      before do
+        config[:use_iam_profile] = false
+      end
+
+      it 'throws an error when aws key is missing' do
+        config[:aws_access_key_id] = ''
+        expect { driver.send(:aws_credentials) }.to raise_error(Kitchen::UserError)
```

Comment:
i think just make it public, there are some other public helpers already in the driver.